src/components/DownloadMenu/useDownloadMenu.js

```diff
@@ -121,6 +120,10 @@ const useDownloadMenu = (relativePeriodDate) => {
             // TODO add common parameters
             // if there are for both event/enrollment and PT/LL
 
+            if (current.outputType !== OUTPUT_TYPE_ENROLLMENT) {
```

Comment:
might be more transparent to look for `=== OUTPUT_TYPE_EVENT` rather than the opposite?

Reply:
The same condition is used in other places.
I believe the reason is because there will be more input types in the future, but I agree it seems a bit premature to think all other types can be treated as events in this specific case.